platform_versioned_docs/version-24.1/enterprise/configuration/configtables/data_features_yml.yml

```diff
@@ -0,0 +1,11 @@
+---
+-
+  tower.yml:            '`tower.content.max-file-size`'
+  Description: >
+    You can set the Data Explorer downloadable file size limit. Increasing this value may degrade Platform performance.
```

Comment:
This value also controls the max file size limit for downloadable workflow reports (which is outside of DataExplorer)